### DISCUSSION TITLE
Failed to load lap data?

### DISCUSSION THREAD

**Discussion by Jak7774** [theOehrly/Fast-F1#533] [Q&A] [answered]
Hey everyone, 
So I am having an issue where I cannot load lap data or telemetry data, how can I resolve this issue? I have tried loading races from 2018 to 2023 and the issue is with all tracks. 

My code is below. I have also tried to use the debug option from another discussion here to find the issue but I am not clear on how to revole this. The debug is attached.
[Debugging.pdf](https://github.com/theOehrly/Fast-F1/files/14446939/Debugging.pdf)

```

from matplotlib import pyplot as plt
import pandas
import fastf1 
from fastf1 import plotting
from fastf1 import utils
import fastf1.legacy

fastf1.plotting.setup_mpl()

fastf1.Cache.enable_cache('cache') 

Load the session data into a session object
session = fastf1.get_session(2018, 1, 'R')
session.load()

```

**Comment by Casper-Guo** [comment, marked as answer]
I cannot reproduce

The main error seems to be here:

```
Traceback (most recent call last):
File "C:\Users\je116\AppData\Roaming\Python\Python311\sitepackages\fastf1\core.py", line 1175, in load
self._load_laps_data(livedata)
File "C:\Users\je116\AppData\Roaming\Python\Python311\sitepackages\fastf1\core.py", line 1284, in _load_laps_data
d2 = self.__fix_tyre_info(d2)
^^^^^^^^^^^^^^^^^^^^^^^^
File "C:\Users\je116\AppData\Roaming\Python\Python311\sitepackages\fastf1\core.py", line 1492, in __fix_tyre_info
for key, value in row.iteritems():
^^^^^^^^^^^^^
File "C:\Users\je116\AppData\Roaming\Python\Python311\sitepackages\pandas\core\generic.py", line 5989, in __getattr__
return object.__getattribute__(self, name)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'Series' object has no attribute 'iteritems'
```

`iteritems` was deprecated in Pandas version 2.0.0+. FastF1 should have addressed this a while back in the 3.0.0 release (IIRC). If you search the codebase now there is no usage of `iteritems`. Advise you update fastf1 and try again.

**Reply by Jak7774** [reply]
Fantastic, that has fixed my issue! I had not realised my fastf1 package was out of date. Thank you! :)
